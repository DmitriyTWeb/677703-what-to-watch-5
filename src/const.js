export const RatingDesc = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

export const DEFAULT_RENDERED_FILMS_COUNT = 8;
export const PLAYING_TIMEOUT = 1000;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`,
};

export const AppRoute = {
  LOGIN: `/login`,
  ROOT: `/`,
  MYLIST: `/mylist`,
  FILMS_ID: `/films/:id`,
  PLAYER_ID: `/player/:id`,
  REVIEW_ID: `/films/:id/review`,
  ERROR: `/error`,
};

export const APIRoute = {
  FILMS: `/films`,
  LOGIN: `/login`,
  LOGOUT: `/logout`,
  PROMO: `/films/promo`,
};
