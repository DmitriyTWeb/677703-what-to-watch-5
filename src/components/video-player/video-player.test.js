import React, {createRef} from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";
import {mockFilm} from "../../const.js";

const videoRef = createRef();
it(`Should VideoPlayer render correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer
        src={mockFilm.video}
        coverImg={mockFilm.coverImg}
        onVideoMount={() => {}}
        videoRef={videoRef}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
