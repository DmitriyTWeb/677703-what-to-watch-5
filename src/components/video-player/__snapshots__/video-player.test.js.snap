// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  height="175"
  muted="muted"
  poster="img/the-grand-budapest-hotel.jpg"
  src="https://some-link"
  width="280"
>
  Please use another web browser with video tag support
</video>
`;
